packages/main/test/specs/RangeSlider.spec.js

```diff
@@ -262,6 +262,557 @@ describe("Testing events", () => {
 	});
 });
 
+
+describe("Accessibility: Testing focus", () => {
+	it("Click anywhere in the  Range Slider should focus the closest handle", () => {
+		browser.url("http://localhost:8080/test-resources/pages/RangeSlider.html");
```

Comment:
This seems to couple the tests with specific environment. Shouldn't that be  at least a relative URL